src/theme/snackbar.m.css

```diff
@@ -0,0 +1,54 @@
+@import '../common/styles/variables.css';
+
+.root {
```

Comment:
not sure if this should all be here or in the dojo theme?

Reply:
@tomdye Good question. I added the minimum structure and positioning to this CSS, just so that it uses flex box and displays at the bottom of the window. Should this not be here?